.github/actions/land-blocking/find-lbt-images.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Copyright (c) The Libra Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+# adapted from --build-all-cti option from libra/docker/build-aws.sh, which is how
+# land blocking test builds images before running cluster-test
+REPOS=(libra_validator libra_cluster_test libra_init libra_safety_rules)
```

Comment:
Any way we could generalize this into one place somehow?
Maybe make `cluster_test_images.sh` and include it both here and in other build-aws?

Reply:
That might get a bit messy, since the list in `build-aws.sh` actually specifies the codebuild project name, which can be different than the actual repo name. I think we can just be mindful about keeping them in sync (I'll add a note in both scripts). I don't expect these images to change too often

Reply:
Ok :)
Let's make a note then.